typescript/best-practice-web/wdio.visual.conf.ts

```diff
@@ -0,0 +1,106 @@
+const sauceOptions = {
+    username: process.env.SAUCE_USERNAME,
+    accesskey: process.env.SAUCE_ACCESS_KEY
+};
+const visualOptions = {
+    apiKey: process.env.SCREENER_API_KEY,
+    projectName: 'Sauce Demo',
+    scrollAndStitchScreenshots: true
+};
+
+exports.config = {
+    // ====================
+    // More information about the configuration is found here https://webdriver.io/docs/configurationfile.html
+    // ====================
+    runner: 'local',
+    specs: [
+        './test/specs/**/visual*.ts'
+    ],
+    maxInstances: 100,
+    logLevel: 'error',
+    bail: 0,
+    baseUrl: 'https://www.saucedemo.com/',
+    waitforTimeout: 10000,
+    connectionRetryTimeout: 90000,
+    connectionRetryCount: 0,
+    framework: 'jasmine',
+    reporters: ['spec'],
+    jasmineOpts: {
+        defaultTimeoutInterval: 60000,
+    },
+    services: [],
+
+    //Screener Configuration
+    hostname: 'hub.screener.io',
```

Comment:
`hostname, port, protocol` are not needed anymore. The Sauce Service will handle that automatically if the `sauce:visual` object is provided. I know it's not in the docs, but it's in the code , see [here](https://github.com/webdriverio/webdriverio/blob/9d2220e89144b0ca69232737957ad5fc32ca1300/packages/webdriverio/src/utils/detectBackend.ts#L82) and [here](https://github.com/webdriverio/webdriverio/blob/9d2220e89144b0ca69232737957ad5fc32ca1300/packages/webdriverio/src/utils/detectBackend.ts#L35)